.golangci.yml

```diff
@@ -7,7 +7,6 @@ run:
 linters:
   enable:
     - bodyclose
-    - deadcode
```

Comment:
These linters are removed as they are deprecated and no longer supported by the latest version of `golangci-lint`.